src/NetSuiteClient.php

```diff
@@ -42,38 +42,40 @@ class NetSuiteClient
      * @param array $options
      * @param SoapClient $client
      */
-    public function __construct($config = null, $options = array(), $client = null)
-    {
-        if ($this->config) {
-            $this->config = $config;
-        } else {
-            self::createFromEnv();
-        }
-        $options = $this->createOptions($this->config, $options);
-        $wsdl = $this->createWsdl($this->config);
-        $this->client = $client ?: new SoapClient($wsdl, $options);
-    }
+     public function __construct($config = null, $options = array(), $client = null)
+     {
+         if ($this->config) {
```

Comment:
This eval should be on `$config` rather than `$this->config`

Reply:
Fixed.